pkg/testframework/imageregistry.go

```diff
@@ -342,3 +342,24 @@ func MustEnsureClusterOperatorStatusIsSet(t *testing.T, client *Clientset) {
 		t.Fatal(err)
 	}
 }
+
+func MustEnsureOperatorIsNotHotLooping(t *testing.T, client *Clientset) {
+	// Allow the operator a few seconds to stabilize
+	time.Sleep(15 * time.Second)
```

Comment:
I wonder if 15 seconds isn't enough (based on the ci infrastructure flaking so much) https://github.com/openshift/cluster-image-registry-operator/pull/159/files#diff-a8d53698d228697ddce3c9426f371e5b
Do you think we could have it check every 5 seconds for a minute with a wait.Poll ?

Reply:
what purpose would wait.poll serve?  wait until we think it's stable, then wait another 15s to confirm it is stable?